src/data/showcases.js

```diff
@@ -1144,6 +1144,14 @@ export const Showcases = [
     source: null,
     tags: ["nftsupport", "wallet"],
   },
+  { 
+    title: "Anvil Agency",
+    description: "Anvil provides advanced solutions for Web3 projects through a variety of services.",
```

Comment:
Fixed in https://github.com/Cardano-Forge/developer-portal/pull/1 but you will have to merge it in your repo since we have no write access to this PR branch.